src/main/scala/com/campudus/tableaux/HelperFunctions.scala

```diff
@@ -28,7 +29,7 @@ object HelperFunctions {
     fromColumn <- notNull(json.getLong("fromColumn"), "fromColumn")
   } yield Some(toTable, toColumn, fromColumn)
 
-  private def checkAndGetColumnInfo(seq: Seq[JsonObject]): ArgumentCheck[Seq[(String, TableauxDbType, Option[LinkConnections])]] = for {
+  private def checkAndGetColumnInfo(seq: Seq[JsonObject]): ArgumentCheck[Seq[(String, TableauxDbType, Option[Ordering], Option[LinkConnections])]] = for {

```

Comment:
Maybe it would help to put the Tupel into a type (case class for example): `(String, TableauxDbType, Option[Ordering], Option[LinkConnections])`